packages/replay/src/main.ts

```diff
@@ -47,8 +47,8 @@ import {
 } from "./types";
 import { ReplayClient } from "./upload";
 import { exponentialBackoffRetry, getDirectory, maybeLog, openExecutable } from "./utils";
-import { Agent as HttpAgent, AgentOptions } from "http";
-import { Agent as HttpsAgent } from "https";
+import { getLaunchDarkly } from "./launchdarkly";
+import { Agent, AgentOptions } from "http";
```

Comment:
Unrelated: Removed unnecessary imported `HttpsAgent`. Renamed to `AgentOptions` to `AgentOptions`.